babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -788,12 +788,64 @@ SqlIndex SqlCreateTableIndex(Span s) :
 SqlNode SqlExecMacro() :
 {
     SqlIdentifier macro;
+    SqlNodeList params = new SqlNodeList(getPos());
     Span s;
 }
 {
     macro = CompoundIdentifier() { s = span(); }
+    [
+        SqlExecMacroArgument(params)
+    ]
+    {
+        return new SqlExecMacro(s.end(this), macro, params);
+    }
+}
+
+void SqlExecMacroArgument(SqlNodeList params) :
+{
+    SqlNode e;
+}
+{
+    <LPAREN>
+    (
+        e = SqlSimpleIdentifierEqualLiteral()
+        {
+            params.add(e);
+        }
+        (
+            <COMMA>
+            e = SqlSimpleIdentifierEqualLiteral()
+            {
+                params.add(e);
+            }
+        )*
+    |
+        e = Literal()
+        {
+            params.add(new SqlExecMacroParam(getPos(), e));
+        }
+        (
+            <COMMA>
+            e = Literal()
+            {
+                params.add(new SqlExecMacroParam(getPos(), e));
+            }
+        )*
+    )
+    <RPAREN>
+}
+
+SqlNode SqlSimpleIdentifierEqualLiteral() :
+{
+    SqlNode name;
```

Comment:
Let's make this the more specific SqlIdentifier type.

Reply:
Change made in https://github.com/googleinterns/calcite/pull/108/commits/e667292f4b74ef3f262506850fcf3cf6b113988e